README.md

```diff
@@ -1,30 +1,60 @@
-# SourceExplorer
-Last updated by | Egor Belekhov | Nov 23, 2021 at 11:23 AM GMT+3
+# GitRunner
+Last updated by | Egor Belekhov | Nov 24, 2021 at 10:03 AM GMT+3
 ***
 
-## Overview
+# GitRunner
 
-We need an application that will reduce the time and simplify the work with git. It should provide the ability to manage all the necessary repositories from one place (change, update branches) as well as build and run the necessary parts of the project.
+## Overview
 
+We need an application that will reduce the time and simplify the work with git. It should provide the ability to manage all the necessary repositories from one place (change, update branches).
 
 ### Use Cases
 
-SourceExplorer should be user to update, build and run microservices. The list of paths to the necessary repositories will be in the configuration file.
-For each of them, it will be possible to see the included branch and change it to another existing branch.  The selected branch can be updated by performing a pull operation. 
-The next option is to select a mod and build the project according to it.
-And lastly launching the project. Also these possibilities should be available for a group of projects, i.e. the selection of settings and then the execution of the necessary logic for all of them.
+The tool should be able to:
+
+1. Synchronize repositories (all or selected). Pull the branch that is currently selected. Selected branch shall be visible to user.
```

Comment:
Please add one more feature: When started, the application will pick up and display all git-repositories in the main git folder specified by the user. The user can select the main folder in Settings.

Reply:
added